bank/persistence/persistence_tests/account_related_test.go

```diff
@@ -0,0 +1,84 @@
+package persistance_test
+
+import (
+	"github.com/iliyanmotovski/bankv3/bank/domain"
+	"github.com/iliyanmotovski/bankv3/bank/persistence"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+	"testing"
+	"time"
+)
+
+func TestItWritesAndReadsAccountDataFromDB(t *testing.T) {
+	session, _ := mgo.DialWithTimeout("localhost:27016", time.Second*10)
+	defer session.Close()
+	session.SetMode(mgo.Monotonic, true)
+
+	sessionStore := persistence.NewMongoSessionStore(*session, "bankTest")
+
+	testAccount := domain.Account{UserID: "1234", Currency: "testC", Amount: 10}
+
+	accountID, err := sessionStore.InsertAccount(testAccount.UserID, testAccount)
+	if err != nil {
+		t.Error(err.Error())
+	}
+
+	a, err := sessionStore.GetAccounts(testAccount.UserID)
+	if err != nil {
+		t.Error(err.Error())
+	}
+	accountFromDB := *a
+
+	if accountFromDB[0].AccountID != accountID || accountFromDB[0].UserID != testAccount.UserID || accountFromDB[0].Currency != testAccount.Currency || accountFromDB[0].Amount != 10 {
+		t.Errorf("expected accountFromDB AccoundID='%s' UserID='%s' Currency='%s' Amount='%f'\n"+
```

Comment:
expected account = %v, got: %v 

you can also implement String() function of account.